components/form-control/script.js

```diff
@@ -6,7 +6,6 @@
 
 import components from 'coherent-gameface-components';
 // eslint-disable-next-line no-unused-vars
-import tooltip from 'coherent-gameface-tooltip';
```

Comment:
Why did we remove the tooltip here? It is used in the `showError` method.

Reply:
Did not know that.